docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -1,8 +1,303 @@
+can optionally include or exclude fields from the analysis. For more information
+about field selection, see the
+{ref}/explain-dfanalytics.html[explain data frame analytics API].
+
+
+[discrete]
+[[dfa-classification-supervised]]
+=== Training the {classification} model
+
+{classification-cap} – just like {regression} – is a supervised {ml} process.
+When you create the {dfanalytics-job}, you must provide a data set that contains
+the _ground truth_. That is to say, your data set must contain the {depvar} 
+and the {feature-vars} fields that are related to it. You can divide the data
+set into training and testing data by specifying a `training_percent`. By
+default when you use the
+{ref}/put-dfanalytics.html[create {dfanalytics-jobs} API], 100% of the 
+<<dfa-classification-field-type-docs-limitations,eligible documents>> in the 
+data set are used for training. If you divide your data set, the job stratifies 
+the data to ensure that both the training and testing data sets contains classes 
+in proportions that are representative of the class proportions in the full data 
+set.
+
+When you are collecting a data set to train your model, ensure that it
+captures information for all of the classes. If some classes are poorly
+represented in the training data set (that is, you have very few data points per 
+class), the model might be unaware of them. In general, complex decision 
+boundaries between classes are harder to learn and require more data points per 
+class in the training data.
+
+////
+It means that you need to supply a labeled training data set that has a {depvar} 
+and some fields that are related to it. The {classification} algorithm learns 
+the relationships between these fields and the {depvar}. Once you’ve trained the 
+model on your training data set, you can reuse the knowledge that the model has 
+learned about the relationships between the data points to classify new data.
+The effects of imbalanced data are automatically mitigated before the 
+training. Nonetheless, it is a good idea to train your model with a data set 
+that is approximately balanced. That is to say, ideally your data set should 
+have a similar number of data points for each class.
+////
+
+
+[discrete]
+[[dfa-classification-algorithm]]
+==== {classification-cap} algorithms
+
+//tag::classification-algorithms[]
+{classanalysis-cap} uses an ensemble algorithm that is a type of boosting called 
+boosted tree regression model which combines multiple weak models into a 
+composite one. It uses decision trees to learn to predict the probability that a 
+data point belongs to a certain class. A sequence of decision trees are trained 
+and every decision tree learns from the mistakes of the previous one. Every tree 
+is an iteration of the last one, hence it improves the decision made by the 
+previous tree.
+//end::classification-algorithms[]
+
+[discrete]
+[[dfa-classification-deploy]]
+==== Deploying the model
```

Comment:
Should the "Deploying the model" section be a peer to the "Training a model" section? As it is, the former is embedded in the latter, which I don't think is correct
```suggestion
=== Deploying the model
```